docker/README.md

```diff
@@ -10,47 +10,46 @@ A Docker **image** is a compressed object holding all the dependencies that you
 
 A **Dockerfile** is how you specify the dependencies in your image. It uses a simplified command structure (see https://docs.docker.com/engine/reference/builder/) which allows you to run any bash command. The nice thing about a Docker image is that it caches the image between every `RUN`, `ADD` or `COPY` command. This allows you to build images quickly after you've built it the first time. 
 
-Our latest Docker images will be stored on Docker Hub here https://hub.docker.com/repository/docker/avsurfer123/c-runtime.
+Our latest Docker images will be stored on Docker Hub here https://hub.docker.com/repository/docker/pierobotics/c-runtime.
 
 
 ## Installation
 
-To install Docker, go here https://docs.docker.com/get-docker/.
+To install Docker, go here https://docs.docker.com/get-docker/. If you're on Windows and don't have Windows 10 Pro, it is highly recommended to install WSL 2 here https://docs.microsoft.com/en-us/windows/wsl/install-win10.
```

Comment:
What does Windows 10 Pro offer that doesn't really need WSL? What does this have to do with Docker? Does this have to do with architecture?

Reply:
Docker Desktop can't be normally installed if you have Windows 10 Home, I think due to HyperV issues. If you have Home, you either need to install Docker Toolbox which has Oracle VirtualBox, which is not recommended, or install WSL 2 to have a Linux VM.